fenix/app/src/main/java/org/mozilla/fenix/home/sessioncontrol/viewholders/PrivateBrowsingDescriptionViewHolder.kt

```diff
@@ -121,3 +137,59 @@ private fun PrivateBrowsingDescriptionPreview() {
         )
     }
 }
+
+/**
+ * Total Private Browsing Mode description.
+ *
+ * @param onLearnMoreClick Invoked when the user clicks on the who can see my activity link.
+ */
+@Composable
+fun FeltPrivacyModeDescription(
+    onLearnMoreClick: () -> Unit,
+) {
+    Card(
+        modifier = Modifier
+            .wrapContentHeight(),
+        shape = RoundedCornerShape(8.dp),
+        backgroundColor = FirefoxTheme.colors.layerAccentNonOpaque,
+        elevation = 0.dp,
+    ) {
+        Column(
+            modifier = Modifier.padding(PaddingValues(16.dp)),
+        ) {
+            Text(
+                text = stringResource(id = R.string.felt_privacy_desc_card_title),
+                style = FirefoxTheme.typography.headline7,
+                color = FirefoxTheme.colors.textPrimary,
+            )
+
+            Spacer(modifier = Modifier.height(8.dp))
+
+            LinkText(
+                text = stringResource(
+                    id = R.string.felt_privacy_desc_card_subtitle,
+                    stringResource(id = R.string.app_name),
+                ),
+                linkTextState = LinkTextState(
+                    text = stringResource(id = R.string.felt_privacy_desc_card_subtitle_link_portion),
+                    url = "",
+                    onClick = { onLearnMoreClick() },
+                ),
+                style = FirefoxTheme.typography.body2.copy(
+                    color = FirefoxTheme.colors.textPrimary,
+                ),
+                linkTextDecoration = TextDecoration.Underline,
+            )
+        }
+    }
+}
+
+@Composable
+@Preview
+private fun FeltPrivacyModeDescriptionPreview() {
+    FirefoxTheme {
+        FeltPrivacyModeDescription(
```

Comment:
Could we add a suitable Column layer background for this Preview? It's just looking a bit dark in the Preview right now without any layer background color behind it.

<img width="472" alt="Screen Shot 2023-09-05 at 1 57 41 PM" src="https://github.com/mozilla-mobile/firefox-android/assets/1190888/0a9377eb-7c71-4b9f-89f8-197317174d33">